python/lsstdesc/pipe/task/coadd_in_cells.py

```diff
@@ -265,3 +256,12 @@ def get_noise_exp(exp, rng):
     noise_exp.setDetector(exp.getDetector())
 
     return noise_exp
+
+
+def hash_function(seed, tract, patch, band):
```

Comment:
This seems fine. The other option is to use something from hashlib (like sha256) applied to a string of bytes extracted from string representations of the hash keys.